editoast/src/views/timetable/stdcm.rs

```diff
@@ -102,8 +111,9 @@ enum StdcmError {
 }
 
 #[derive(Debug, Default, Clone, Serialize, Deserialize, IntoParams, ToSchema)]
-pub(in crate::views) struct InfraIdQueryParam {
+pub(in crate::views) struct StdcmQueryParams {
     infra: i64,
+    return_debug_payloads: Option<bool>,
```

Comment:
suggestion (not tested):

```suggestion
	#[schema(required = false)]
	#[serde(default)]
    return_debug_payloads: bool,
```

if this works that'd avoid the `unwrap_or(false)` below.

(feel free to resolve)

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/13078/commits/7156bc4352d354fecca75764620d65a28507571a